src/components/common/SafeTokenWidget/index.tsx

```diff
@@ -44,10 +44,7 @@ const SafeTokenWidget = () => {
       }
     : undefined
 
-  const safeBalance = balances.balances.items.find((balanceItem) => balanceItem.tokenInfo.address === tokenAddress)
-
-  const safeBalanceDecimals = Number(safeFormatUnits(safeBalance?.balance || 0, safeBalance?.tokenInfo.decimals))
-  const flooredSafeBalance = formatAmountWithPrecision(safeBalanceDecimals, 2)
+  const flooredSafeBalance = formatVisualAmount(allocation, DECIMALS)
```

Comment:
Don't we want to keep the 2 decimals precision? It broke the tests

Reply:
It's the default, and it's not what broken them.

Reply:
@DiogoSoaress you were actually right, it needs the precision, as Franco also pointed out. I've passed it explicitly.